android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1547,7 +1576,44 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         currentBanner = (change.equals("loaded")) ? "suggestion" : "blank";
         RelativeLayout.LayoutParams params = getKeyboardLayoutParams();
         SystemKeyboard.setLayoutParams(params);
+      } else if (url.indexOf("suggestPopup") >= 0) {
+        int start = url.indexOf("pos=") + 4;
```

Comment:
This section of code now appears outdated?

Reply:
Right... guess who forgot to duplicate the in-app keyboard class changes to the system keyboard class...